src/wot-adapter.ts

```diff
@@ -189,14 +204,21 @@ export class WoTAdapter extends Adapter {
 
       wcd.add({ url, authentication: authdata });
       await wcd.save();
-
+      this.handleDeviceAdded(device);
       return device;
     }
   }
 
   handleDeviceSaved(_deviceId: string, _device: DeviceWithoutIdSchema): void {
     const device: WoTDevice = this.getDevice(_deviceId) as WoTDevice;
     device && device.start();
```

Comment:
You can do this in typescript:
```suggestion
    device?.start();
```

Reply:
Or better: move it into the if block :grimacing:

Reply:
Oh sure, sorry I missed it 👍🏻